### DISCUSSION TITLE
Client Certificate thumbprint coming null from the current request context

### DISCUSSION THREAD

**Discussion by navichandan** [CoreWCF/CoreWCF#1186] [General]
We required to add certificate thumbprint verification and we are not able to do so as the object, OperationContext.Current.ServiceSecurityContext is coming as null.  Is  there anyway we get this, we need to get the thumbprint and do the validation.

Please help if any way we get this.

**Comment by shibdhara** [comment]
Any update on the above issue ? can you please confirm will it support in core wcf or not ?
